bin/bump_version.py

```diff
@@ -33,16 +33,19 @@ def bump_version():
     current_version = cibuildwheel.__version__
 
     try:
-        commit_date_str = subprocess.run([
-            'git',
-            'show', '-s', '--pretty=format:%ci',
-            f'v{current_version}^{{commit}}'
-        ], check=True, capture_output=True, encoding='utf8').stdout
+        commit_date_str = subprocess.run(
+            ['git', 'show', '-s', '--pretty=format:%ci', f'v{current_version}^{{commit}}'],
+            check=True,
+            capture_output=True,
+            encoding='utf8',
+        ).stdout
         commit_date_parts = commit_date_str.split(' ')
 
-        url = 'https://github.com/joerick/cibuildwheel/pulls?' + urllib.parse.urlencode({
-            'q': f'is:pr merged:>{commit_date_parts[0]}T{commit_date_parts[1]}{commit_date_parts[2]}',
-        })
+        url = 'https://github.com/joerick/cibuildwheel/pulls?' + urllib.parse.urlencode(
+            {
+                'q': f'is:pr merged:>{commit_date_parts[0]}T{commit_date_parts[1]}{commit_date_parts[2]}',
+            }
+        )
```

Comment:
This is worse, IMO.

Reply:
Why? It's_ really_ easy to miss the `{}` in the original version. Black ensures you don't "helpfully" hide things like this. I'm strongly in favor of the rewritten version, actually, since if I'm reading quickly, I get that it's a set, while in the original version, I'd have to look very carefully.

Reply:
Is it so important that it's a dict? `'q': ` should also clarify that, though?

Reply:
I missed that it was a dict, I thought it was a set. So probably no? :P

Reply:
:-D Black doesn't help there, though, but I guess that's besides the point.